djangoProject/templates/users/users_detail.html

```diff
@@ -0,0 +1,25 @@
+{% extends "users/users_list.html" %}
```

Comment:
It is formally not correct to inherit detail from list. It is better to create some generic template for them